client/state/comments/actions.js

```diff
@@ -178,14 +181,16 @@ export const writeComment = ( commentText, siteId, postId ) => ( {
  * @param {Number} siteId site identifier
  * @param {Number} postId post identifier
  * @param {Number} parentCommentId parent comment identifier
+ * @param {Boolean} query Forces requesting a fresh copy of comments with these query parameters.
```

Comment:
Maybe name this as `refreshCommentListQuery` since we're doing something quite specific

Reply:
Fun fact: I started working on this as a `{Boolean} forceRefresh` flag.
Then I replaced it with a query object (because it doesn't make much sense to have both a flag and the object) and missed the `{Boolean}` leftover.